api/authFunctions.ts

```diff
@@ -0,0 +1,13 @@
+export async function getWeeklyPicks() {
+    try {
+        return await fetch(`https://cloud.appwrite.io/v1/databases/${process.env.DATABASE_ID}/collections/${process.env.WEEKLY_PICKS_COLLECTION_ID}/documents`, {
```

Comment:
@vmaineng Is this the proper way to call Appwrite's API?

Reply:
I replaced this with the proper way to call Appwrite's API, using the Appwrite's in-built SDK method.